apps/web/src/views/FixedStaking/hooks/useHandleWithdrawSubmission.ts

```diff
@@ -25,7 +25,23 @@ export function useHandleWithdrawSubmission({
   const { t } = useTranslation()
   const { toastSuccess, toastInfo } = useToast()
   const { callWithGasPrice } = useCallWithGasPrice()
-  const { fetchWithCatchTxError, loading: pendingTx } = useCatchTxError()
+  const { fetchWithCatchTxError, loading: pendingTx } = useCatchTxError({
+    throwCustomeError: () => {
```

Comment:
As mentioned above, better wrap `throwCustomeError` in `useMemo` for optimization